### DISCUSSION TITLE
[MV3 / uBOL] Skip review for eligible changes to extensions

### DISCUSSION THREAD

**Discussion by hadiskeini** [uBlockOrigin/uBOL-home#217] [Q&A] [answered]
https://developer.chrome.com/blog/extensions-skip-review-eligible-changes

I searched the WWW and this repository and couldn't find any info on whether this change to the Chrome Web Store review process could benefit the uBOL extension and allow it to more frequently update its filter lists.

Does anyone have insights on this?

**Comment by gorhill** [comment, marked as answer]
This doesn't benefit uBOL, or any other content blocker in the same realm. Changes to filter lists -- which uBOL compiles -- are most likely to contain changes to scriptlet injection filters, cosmetic filters, and/or DNR rules outside "safe static rules" which are defined as _"rules with an action of `block`, `allow`, `allowAllRequests` or `upgradeScheme`"_.

Although the policy sounds nice in theory, in practice it's of no benefit to uBOL or other content blockers in the same class, i.e. those doing more than just using `block`/`allow`/`allowAllRequests`, which is essentially all top content blockers.
